### PR TITLE
Delete artifacts from new gui

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qiita_pet/handlers/api_proxy/artifact.py [hunk 1]
```diff
@@ -84,7 +84,8 @@ def artifact_summary_get_request(user_id, artifact_id):
 
     buttons = []
     btn_base = (
-        '<button onclick="set_artifact_visibility(\'%s\', {0})" '
+        '<button onclick="if (confirm(\'Are you sure you want to %s '
+        'artifact id: {0}?\')) {{ set_artifact_visibility(\'%s\', {0}) }}" '
```
Comment (wasade) [inline qiita_pet/handlers/api_proxy/artifact.py]: are the jinja templates dynamic?

Comment (wasade) [inline qiita_pet/handlers/api_proxy/artifact.py]: (confused with the `{{ ... }}`)

Comment (antgonza) [inline qiita_pet/handlers/api_proxy/artifact.py]: Yes, they are dynamic and the {{ is to scape  one {

**Comment by josenavas** [comment]
:+1:

**Pull request merged**